anchor-reference-server/src/main/java/org/stellar/anchor/reference/service/CustomerService.java

```diff
@@ -298,6 +315,14 @@ public ProvidedCustomerField createBankAccountNumberProvidedField() {
     return field;
   }
 
+  public ProvidedCustomerField createBankAccountTypeProvidedField() {
+    ProvidedCustomerField field = new ProvidedCustomerField();
+    field.setType("string");
+    field.setDescription("bank account type of the customer");
+    field.setStatus(Customer.Status.ACCEPTED.toString());
+    return field;
+  }
```

Comment:
Same goes here, lets add the `choices` list.

Reply:
Added